Problem 1.java

```diff
@@ -0,0 +1,22 @@
+// Problem 1: 
+
+// Time Complexity : O(n^2)
+// Space Complexity : O(1)
+// Did this code successfully run on Leetcode : Yes
+// Any problem you faced while coding this : I believe it can be further optimized
```

Comment:
normal sorting itself takes only nlogn. Try the optimized approach to be discussed in class